events/generate/events_generate.go

```diff
@@ -33,8 +34,9 @@ type EventData struct {
 }
 
 type Event struct {
-	Input  EventData
-	Output EventData
+	Input  			EventData
+	Output 			EventData
+	ConvertedOutput EventData
```

Comment:
Formatting. Run goimports?

Reply:
Oh yeah, didn't catch it because this is in the template. I can update.